crates/uv-dev/src/main.rs

```diff
@@ -66,7 +64,6 @@ enum Cli {
     /// cargo run --bin uv-dev -- resolve-many scripts/popular_packages/pypi_10k_most_dependents.txt
     /// ```
     ResolveMany(ResolveManyArgs),
-    InstallMany(InstallManyArgs),
```

Comment:
Does anyone care about preserving this? I can migrate it but I wasn't sure if it was worth it. \cc @konstin

Reply:
It was nice to have but honestly not if its effort to maintain, i don't run it anymore

Reply:
Haha. If I had my way I would remove them all.